cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -249,6 +262,18 @@ if (ORC_PACKAGE_KIND STREQUAL "conan")
   add_resolved_library (orc_zstd ${zstd_LIBRARIES} ${zstd_INCLUDE_DIR})
   list (APPEND ORC_SYSTEM_DEPENDENCIES ZSTD)
   list (APPEND ORC_INSTALL_INTERFACE_TARGETS "$<INSTALL_INTERFACE:$<IF:$<TARGET_EXISTS:zstd::libzstd_shared>,zstd::libzstd_shared,zstd::libzstd_static>>")
+elseif (ORC_PACKAGE_KIND STREQUAL "vcpkg")
+  find_package(zstd CONFIG REQUIRED)
+  add_library(orc_zstd INTERFACE)
+  if (TARGET zstd::libzstd_shared)
+    target_link_libraries(orc_zstd INTERFACE zstd::libzstd_shared)
+  elseif (TARGET zstd::libzstd_static)
+    target_link_libraries(orc_zstd INTERFACE zstd::libzstd_static)
+  else()
+    message(FATAL_ERROR "No suitable ZSTD library target found.")
```

Comment:
This section is not consistent with others. Could we simply use `target_link_libraries(orc_zstd INTERFACE $<IF:$<TARGET_EXISTS:zstd::libzstd_shared>,zstd::libzstd_shared,zstd::libzstd_static>)`

Reply:
Sure, updated.